server/svix-server/src/v1/endpoints/event_type.rs

```diff
@@ -280,7 +280,7 @@ async fn get_event_type(
 ) -> Result<Json<EventTypeOut>> {
     let mut query = eventtype::Entity::secure_find_by_name(org_id, evtype_name);
     if let permissions::AllowedFeatureFlags::Some(flags) = feature_flags {
-        query = query.filter(eventtype::Column::FeatureFlag.is_in(flags.into_iter()));
+        query = eventtype::Entity::filter_feature_flags(query, flags);
```

Comment:
AFAICT we haven't really done anything like this, but FWIW I like it. It's certainly more composable than the individual methods on the Entity.